packages/vuetify/src/components/VExpansionPanel/VExpansionPanel.sass

```diff
@@ -184,3 +189,14 @@
 
     &--active
       max-width: $expansion-panel-inset-active-max-width
+
+.v-expansion-panels--flat
+  .v-expansion-panel
```

Comment:
These styles will cascade to the inner panel, see https://github.com/vuetifyjs/vuetify/issues/9692 / https://github.com/vuetifyjs/vuetify/pull/9696

Reply:
yup, pushed an update